app/models/reports/result.rb

```diff
@@ -179,8 +179,7 @@ def calculate_period_info(range)
       self.period_info = range.each_with_object({}) do |period, hsh|
         hsh[period] = {
           name: period.to_s,
-          start_date: period.start_date.to_s(:day_mon_year),
-          ltfu_since_date: period.start_date.advance(months: -12).to_s(:day_mon_year),
```

Comment:
We don't need a `start_date` anymore. We only show end of month numbers everywhere now!